guides/data_science/deep_learning/capsule_net.md

```diff
@@ -0,0 +1,19 @@
+CNN’s detect features in images and learn how to recognize objects with this information. Layers near the start detecting really simple features like edges and curves, layers in the middle detect a little more complex stuff like quadrilaterals and polygons while layers that are deeper can detect even more complex features like a person's mouth or his/her face. It then uses all this information it learned, to make a final prediction.
```

Comment:
What is a CNN? Please explain a bit about CNNs before you embark on Capsule-Nets